### PR TITLE
Fix VCL syntax

### PR DESCRIPTION
**Pull request by zipkid**
#### Pull Request (PR) description

Fix VCL syntax

https://varnish-cache.org/docs/3.0/reference/vcl.html?highlight=timeout#backend-declarations

Error:
Message from VCC-compiler:
Expected ID got ';'
(program line 346), at
('/etc/varnish/includes/backends.vcl' Line 7 Pos 23) .connect_timeout = 5;
----------------------#-

### CODE REVIEW

**Comment by evgeni** [comment]
did the syntax change, or did people just always pass `123s` strings to that param before?

https://github.com/voxpupuli/puppet-varnish/commit/7b0a649eb9e28bcb1e1b52f6b99445a49b8f72b2 is where it was originally added, without any typing :(

**Comment by zipkid** [comment]
@evgeni There used to be no type check on the timeout parameters so people just added the string with trailing 's' indeed.

**Comment by evgeni** [comment]
In https://github.com/voxpupuli/puppet-varnish/commit/c408cbe486442b09359b7da5d873a836b3d868ff we allowed Strings again, so people *can* pass `5s` and with this change they would get `5ss` (granted, if they pass a `5` today, it won't work, and that's the only allowed thing in the *released* 3.0.0 version).

I guess the question is "should we allow `5s` and somehow catch that, or ignore and only allow a "numeric string".

I bet @voxel01 has an opinion as the author of https://github.com/voxpupuli/puppet-varnish/commit/c408cbe486442b09359b7da5d873a836b3d868ff

**Comment by voxel01** [comment]
As also for example 5m would be valid from Varnish side. A own Type would be great. 
you are right plain Int without handling would be false there.

File: templates/includes/backends.vcl.erb
Comment (voxel01) [inline templates/includes/backends.vcl.erb:1]: I would recommend doing the decision (Integer + 's' vs. String) in the template.

**Review by voxel01 — APPROVED** [review]
Thanks well done!

**Pull request merged**